addons/account/wizard/accrued_orders.py

```diff
@@ -0,0 +1,116 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models, fields, api, _, Command
+from odoo.tools import format_date, float_is_zero
+from odoo.exceptions import UserError
+from dateutil.relativedelta import relativedelta
+import json
```

Comment:
Imports should be sorted according to guidelines (https://www.odoo.com/documentation/14.0/developer/misc/other/guidelines.html#imports) ;).